### DISCUSSION TITLE
Bar-based status design

### DISCUSSION THREAD

**Discussion by iamtakingiteasy** [upptime/upptime#504] [Ideas]
It would be nice if there was an option to display individual checks status for configurable past periods as vertical bars indicating individual check result.

Example: https://discordstatus.com/

Example: ![example](https://user-images.githubusercontent.com/1336778/134628766-a3fe0981-0926-4285-ab46-891a21c3e4cb.png)

**Comment by aschenmaker** [comment]
Yes. Github status is also awesome. 
> https://www.githubstatus.com/
